lib/server.js

```diff
@@ -133,16 +134,27 @@ function createServer(options, clients, name) {
         name: common.METRIC_REQUEST_COUNTER,
         help: 'count of Muskie requests completed'
     });
-    // A histogram to track the time to first byte.
+
+    /*
+     * A log-linear histogram to track the time to first byte.
+     * Track values between 10^(-4) and 10^(4) seconds.
```

Comment:
Same comment I made in https://github.com/joyent/manta-muskie/pull/47#discussion_r368668386 applies here. I think we should either fix the comment or the metric.